### PR TITLE
RELATED: ONE-3387 Fix color configuration for pie and donut chart

### PR DESCRIPTION
**Pull request by peterlacko**

### CODE REVIEW

File: stories/core_components/PieChart.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ import {
     DATA_LABELS_HIDDEN_CONFIG,
     DATA_LABELS_AUTO_CONFIG
 } from '../data/configProps';
+import { CUSTOM_COLOR_PALETTE } from '../data/colors';
```
Comment (ivan-nejezchleb) [inline stories/core_components/PieChart.tsx]: use import { CUSTOM_COLOR_PALETTE_CONFIG } from '../data/configProps'; instead

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1779/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1259/ : SUCCESS in 1m 47s

**Comment by peterlacko** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1780/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1260/ : SUCCESS in 2m 07s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/735/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1782/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/393/ : UNSTABLE in 13m 04s

**Comment by lhorak** [comment]
Reopen to 6.1.0 please if needed

**Comment by gdgate** [comment]
Build failed (check-teardown pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-stop-zuul-docker/214/ : FAILURE in 4m 02s

**Pull request closed without merging**